style.css

```diff
@@ -70,10 +71,6 @@ p:hover {
   width: 100%;
 }
 
```

Comment:
- [ ] [OPTIONAL] There is an empty space on the right side of your page. The issue is arising because of an overflow happening in the About section. Please fix it. 😎️

<img src="https://user-images.githubusercontent.com/17243325/215773998-d3c30d45-2413-47b3-ad1a-b36a90c2418f.png" width="300">